kafka-ui-api/src/test/java/com/provectus/kafka/ui/KafkaConsumerTests.java

```diff
@@ -126,4 +127,29 @@ public void shouldReturn404ForNonExistingTopic() {
         .expectStatus()
         .isNotFound();
   }
+
+  @Test
+  public void shouldRetrieveConfig() {
+    var topicName = UUID.randomUUID().toString();
+
+    webTestClient.post()
+            .uri("/api/clusters/{clusterName}/topics", LOCAL)
+            .bodyValue(new TopicCreation()
+                    .name(topicName)
+                    .partitions(1)
+                    .replicationFactor(1)
+                    .configs(Map.of())
+            )
+            .exchange()
+            .expectStatus()
+            .isOk();
+
+    WebTestClient.ListBodySpec result = webTestClient.get()
+            .uri("/api/clusters/{clusterName}/topics/{topicName}/config", LOCAL, topicName)
+            .exchange()
+            .expectStatus()
+            .isOk()
+            .expectBodyList(TopicConfig.class);
```

Comment:
Could you please validate the actual response with an expected one including `sourceConfig`, `isSensitive` and `isReadOnly` properties

Reply:
Hey @IldarAlmakaev , I updated the tests